src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -1054,6 +1071,29 @@ protected override void ProcessRecord()
             }
         } // ProcessRecord
 
+        private string ReplacePathWithLastLocationIfApplicable(string path)
+        {
+            Environment.SetEnvironmentVariable("DEBUG_PATH", Path);
```

Comment:
I don't think we want this.

Reply:
True. Well spotted. This was a development debugging leftover, sorry.